internal/clusterfeature/service.go

```diff
@@ -84,18 +73,25 @@ func (s FeatureService) Details(ctx context.Context, clusterID uint, featureName
 
 	// TODO: check cluster ID?
 
-	logger.Debug("retrieving feature from repository")
-	feature, err := s.featureRepository.GetFeature(ctx, clusterID, featureName)
+	logger.Debug("retrieving feature manager")
+	featureManager, err := s.featureManagerRegistry.GetFeatureManager(featureName)
 	if err != nil {
-		const msg = "failed to retrieve feature from repository"
+		const msg = "failed to retrieve feature manager"
```

Comment:
What's the added value of a constant here?

Reply:
It's defined once and used by both the debug log and the error wrapping.

Reply:
Yeah...it should be gone. But not in this PR. I don't see how this is useful other than being a noise.